### DISCUSSION TITLE
Blog: Architecture of our Application

### DISCUSSION THREAD

**Discussion by qvalentin** [FileFighter/filefighter.github.io#7] [General]
Hey,
comment your feedback about our "Architecture of our Application" page from your peer-review here:

**Comment by DSinMeliodas** [comment]
Dear FileFighter Team,
First of all I want to say that your blog has improved comparing to the last time.
Your blog is very well structured and very detailed. The pictures you use are quiet good and support your explanations very well, but to do better you could include the graphics directly in your explanations.

With best reagards,
The Futurum Team
